metering/configuring_metering/metering-common-config-options.adoc

```diff
@@ -82,22 +87,81 @@ spec:
     spec:
       nodeSelector:
-        "node-role.kubernetes.io/infra": "true"
+        "node-role.kubernetes.io/infra": "" <1>
 
   presto:
     spec:
       coordinator:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": "" <1>
       worker:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": "" <1>
   hive:
     spec:
       metastore:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": "" <1>
       server:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": "" <1>
+----
+<1> Add a `nodeSelector` parameter with the appropriate value to the component you want to move. You can use a `nodeSelector` in the format shown or use key-value pairs, based on the value specified for the node.
+
+[NOTE]
+====
+Add the `openshift.io/node-selector: ""` namespace annotation to the metering namespace YAML file before configuring specific node selectors for the operand Pods. When the `openshift.io/node-selector` annotation is set on the project, the value is used in preference to the value of the `spec.defaultNodeSelector` field in the cluster-wide Scheduler object.
+====
+
+.Verification
+You can verify the metering node selectors by performing any of the following checks:
+
+*  Verify that all Pods for metering are correctly scheduled on the IP of the node that is configured in the MeteringConfig custom resource:
++
+--
+.Procedure
+.  Check all pods in the `openshift-metering` namespace:
++
+----
+$ oc --namespace openshift-metering get pods -o wide
+----
++
+The output shows the `NODE` and corresponding `IP` for each Pod running in the `openshift-metering` namespace:
++
+----
+NAME                                  READY   STATUS    RESTARTS   AGE     IP            NODE                                         NOMINATED NODE   READINESS GATES
+hive-metastore-0                      1/2     Running   0          4m33s   10.129.2.26   ip-10-0-210-167.us-east-2.compute.internal   <none>           <none>
+hive-server-0                         2/3     Running   0          4m21s   10.128.2.26   ip-10-0-150-175.us-east-2.compute.internal   <none>           <none>
+metering-operator-964b4fb55-4p699     2/2     Running   0          7h30m   10.131.0.33   ip-10-0-189-6.us-east-2.compute.internal     <none>           <none>
+nfs-server                            1/1     Running   0          7h30m   10.129.2.24   ip-10-0-210-167.us-east-2.compute.internal   <none>           <none>
+presto-coordinator-0                  2/2     Running   0          4m8s    10.131.0.35   ip-10-0-189-6.us-east-2.compute.internal     <none>           <none>
+reporting-operator-869b854c78-8g2x5   1/2     Running   0          7h27m   10.128.2.25   ip-10-0-150-175.us-east-2.compute.internal   <none>           <none>
+----
```

Comment:
This shouldn't block this from merging, but we should get a better example as all the Pods are being scheduled to worker nodes (which is the default behavior). We can make a ticket on the metering board for a better example.